expyfun/analyze/_viz.py

```diff
@@ -98,46 +98,49 @@ def barplot(h, axis=-1, ylim=None, err_bars=None, lines=False,
     Parameters
     ----------
     h : array-like
-        If ``h`` is 2-dimensional, heights will be calculated as means along
-        the axis given by ``axis``. If ``h`` is of lower dimension, it is
-        treated as raw height values. If ``h`` is a pandas ``DataFrame`` and
-        ``bar_names`` is None, ``bar_names`` will be inferred from the
-        ``DataFrame``'s ``column`` labels (if ``axis=0``) or ``index`` labels.
+        If `h` is 2-dimensional, heights will be calculated as means along
+        the axis given by `axis`. If `h` is of lower dimension, it is
+        treated as raw height values. If `h` is a pandas ``DataFrame`` and
+        `bar_names` is ``None``, `bar_names` will be inferred from the
+        ``DataFrame``'s `column` labels (if ``axis=0``) or `index` labels.
     axis : int
         The axis along which to calculate mean values to determine bar heights.
-        Ignored if ``h`` is 0- or 1-dimensional.
+        Ignored if `h` is 0- or 1-dimensional.
     ylim : tuple | None
-        y-axis limits passed to ``matplotlib.pyplot.subplot.set_ylim()``.
+        y-axis limits passed to ``matplotlib.pyplot.subplot.set_ylim``.
```

Comment:
do you know if we can do single-backticks for functions from other modules? I think it might actually work since sphinx should know about those other modules. But maybe it only works for numpy and scipy.